support-frontend/assets/pages/contributions-landing/components/SetPasswordForm.jsx

```diff
@@ -78,7 +81,7 @@ function onSubmit(props: PropTypes): Event => void {
   return (event) => {
     props.setPasswordHasBeenSubmitted();
     event.preventDefault();
-    trackComponentClick('set-password');
+    trackComponentClick(`set-password-${props.contributionType}`);
```

Comment:
Is it helpful to separate monthly and annual? I guess this can all be joined up again as 'recurring' during data analysis